src/Tools/CommonNetCoreReferences/project.json

```diff
@@ -34,6 +34,8 @@
         "System.Runtime.InteropServices": "4.3.0",
         "System.Runtime.Loader": "4.3.0",
         "System.Runtime.Numerics": "4.3.0",
+        "System.Runtime.Serialization.Primitives": "4.3.0",
+        "System.Runtime.Serialization.Json": "4.3.0",
```

Comment:
BTW, this https://github.com/dotnet/roslyn/pull/15658 is needed to run scripts on Core 1.1